frontend/main.js

```diff
@@ -109,17 +109,26 @@ async function startApp () {
     sbp('gi.ui/seriousErrorBanner', error)
     if (error?.name === 'ChelErrorForkedChain') {
       const rootState = sbp('state/vuex/state')
-      const type = rootState.contracts[contractID].type || '(unknown)'
+      if (!rootState.contracts[contractID]) {
+        console.error('Forked chain detected. However, there is no contract entry.', { contractID }, error)
+        return
```

Comment:
Can you add a comment explaining here why we don't proceed with recovery? Is it because in this situation we're no longer subscribed...? But then how did we get the error if we weren't subscribed...?